static/authui/css/main.css

```diff
@@ -558,26 +577,27 @@ a.btn {
 }
 
 .settings-identity-identity-section .icon {
+  align-self: start;
+  justify-self: center;
   grid-area: icon;
-  background-repeat: no-repeat;
-  background-size: 24px 24px;
-  background-position: center;
-  width: 24px;
-  height: 24px;
-  margin: 6px;
+  margin: 2px;
 }
 
 .settings-identity-identity-section .name {
   grid-area: name;
   align-self: center;
-  justify-self: start;
+  /* We must keep justify-self: stretch */
```

Comment:
`justify-self: stretch` is the default value. So I omitted it.

Reply:
Oh I see. How about saying 'We must keep using the default value justify-self: stretch' instead, since it's rather confusing to see 'we must keep' but the thing to keep is nowhere to found?